### PR TITLE
Update build-macos.md

### PR DESCRIPTION
**Pull request by ghost**
Cleaned formatting, grammar, and reorder. Added item 6-testnet items

### CODE REVIEW

**Comment by sedited** [comment]
Why are you removing the markdown formatting for code highlighting?

File: doc/build-macos.md [hunk 1]
```diff
@@ -18,18 +18,18 @@ Then install [Homebrew](https://brew.sh).
...
-    brew install librsvg
+        brew install librsvg
 
 NOTE: Building with Qt4 is no longer supported. Build with Qt5.
 
 Build Gridcoin
 --------------
 
-1. Clone the Gridcoin source code and cd into `Gridcoin-Research`
+1. Clone the Gridcoin source code and cd into /Gridcoin-Research.
```
Comment (sedited) [inline doc/build-macos.md]: `Gridcoin-Research` is fine, `/Gridcoin-Research` will try to `cd` into a root directory.

File: doc/build-macos.md [hunk 2]
```diff
@@ -59,10 +59,15 @@ Build Gridcoin
 
         make check
 
-4.  You can also create a .dmg that contains the .app bundle (optional):
+4. To have terminal give full readout if desired: 
 
-        make deploy
+        make V=1 -j #number_of_cores_whatever >& build.log
```
Comment (sedited) [inline doc/build-macos.md]: No need to make again, you run the make command on line 49 in this guide. Add the line 

    To have terminal give full readout if desired:

On line 50.

**Comment by ghost** [comment]
@TheCharlatan I incorporated your good suggestions. Also used a proper editor instead of performing directly on the edit page, which was giving code highlighting issues.

**Pull request merged**